src/c++/perf_analyzer/infer_context.cc

```diff
@@ -118,6 +123,8 @@ InferContext::SendRequest(const uint64_t request_id, const bool delayed)
       it->second.start_time_ = std::chrono::system_clock::now();
       it->second.sequence_end_ = infer_data_.options_->sequence_end_;
       it->second.delayed_ = delayed;
+      thread_stat_->request_send_times_.push_back(
```

Comment:
These don't need to be accurate to the nanosecond. It should be safe for a single call at the start of the function instead of inside each if/else.

Reply:
Moved to start 👍